gtfs-realtime/spec/en/service-alerts.md

```diff
@@ -24,6 +24,8 @@ Entities are selected using their GTFS identifiers, and you can select any of th
 *   Trip - affects a particular trip
 *   Stop - affects a particular stop
 
+You may include multiple entity selectors. When multiple entities are selected in one Alert feed entity, they should be interpreted as being joined by the `AND` logical operator. If you would like to join the entities by the `OR` operator, you should include them in separate Alert feed entities.
```

Comment:
> You may include multiple entity selectors.  When multiple entities are selected in one Alert feed entity, they should be interpreted as being joined by the `AND` logical operator.

->

>You may include multiple entity selectors to apply the same Alert to multiple entities.  When multiple field are included in one `informed_entity`, they should be interpreted as being joined by the `AND` logical operator.

We don't want to duplicate the entire Alert - instead producers can include multiple `informed_entity` clauses to get the `OR` behavior.  See http://ridepsta.net/gtfsrt/alerts?debug for an example alert with multiple  `informed_entity` clauses.

Reply:
After `...by the `AND` logical operator.`, I'd add another short explanation and example, like:

>In other words, the alert should only be applied in context that meet all of the provided `informed_entity` fields. For example, if `route_id: "1"` and `stop_id: "5"` are both included in one `informed_entity`, then the alert should apply only to route 1 at stop 5.  It should NOT be applied to any other stops on route 1, and it should NOT be applied to any other routes at stop 5.